tests/features/browserstack-local.js

```diff
@@ -1,11 +1,11 @@
-/*eslint-env node */
-var Cucumber = require('cucumber');
+/*eslint-env node */
+var Cucumber = require('cucumber');
 var assert = require('cucumber-assert');
 var webdriver = require('selenium-webdriver');
-
-Cucumber.defineSupportCode(function({setDefaultTimeout}) {
-  setDefaultTimeout(60 * 1000);
-});
+
+Cucumber.defineSupportCode(function({setDefaultTimeout}) {

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->